src/client/app/containers/view.mode.ts

```diff
@@ -45,6 +53,27 @@ export class ViewMode implements OnInit, OnDestroy {
         return environment.current.host.toLowerCase();
     }
 
+    get isGist() {
+        return this.snippet && this.snippet.gist;
+    }
+
+    get openInPlaygroundSupported() {
+        let host = environment.current.host.toUpperCase();
+        return host === HostType.EXCEL || host === HostType.WORD || host === HostType.POWERPOINT;
+    }
+
+    get openInHostString() {
+        return this.strings.openInHost.replace('{0}', environment.current.host);
+    }
+
+    get downloadAsHostFileString() {
+        return this.strings.downloadAsHostFile.replace('{0}', environment.current.host);
+    }
+
+    get urlString() {
+        return `URL: ${window.location.href}`;
```

Comment:
I think the URL at the bottom should be the same thing that I could use to embed the snippet into another place, or to open it full-screen in view mode.  So I think this is correct.  It's the *github* URL that should take you to the GitHub page (which it also does).  So I think we're good on both fronts.